extensions/web-audit/package.json

```diff
@@ -50,18 +53,20 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.47.3",
-    "node-fetch": "^3.3.0"
+    "@raycast/api": "^1.94.0"
   },
   "devDependencies": {
-    "@types/node": "18.8.3",
-    "@types/react": "18.0.9",
-    "@typescript-eslint/eslint-plugin": "^5.0.0",
-    "@typescript-eslint/parser": "^5.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "prettier": "^2.5.1",
-    "typescript": "^4.4.3"
+    "@eslint/eslintrc": "^3.3.0",
+    "@eslint/js": "^9.22.0",
+    "@types/node": "22.13.10",
+    "@types/react": "19.0.12",
+    "@typescript-eslint/eslint-plugin": "^8.27.0",
+    "@typescript-eslint/parser": "^8.27.0",
```

Comment:
These are unnecessary. You can refer to this file: https://github.com/raycast/extensions/blob/main/examples/todo-list/package.json to update your dependencies.